### DISCUSSION TITLE
Compatibility with psr/http-client:^0.1?

### DISCUSSION THREAD

**Discussion by angelcabanilles** [PHPOffice/PhpSpreadsheet#1782] [General]
I see that due the recent update [1.16.0] the project now requires psr/http-client:1.0 or higher to work. 

I was wondering if you are you planning on releasing a compatibility version with psr/http-client:^0.1

**Comment by PowerKiKi** [comment]
https://github.com/php-fig/http-client/releases/tag/1.0.0 is almost 3 years old and is the first stable release of the package. We have no plan to support anything earlier than that.
